packages/harfbuzz/build.sh

```diff
@@ -2,20 +2,21 @@ TERMUX_PKG_HOMEPAGE=https://www.freedesktop.org/wiki/Software/HarfBuzz/
 TERMUX_PKG_DESCRIPTION="OpenType text shaping engine"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=7.3.0
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION="8.4.0"
 TERMUX_PKG_SRCURL=https://github.com/harfbuzz/harfbuzz/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=7cefc6cc161e9d5c88210dafc43bc733ca3e383fd3dd4f1e6178f81bd41cfaae
-TERMUX_PKG_AUTO_UPDATE=false
+TERMUX_PKG_SHA256=9f1ca089813b05944ad1ce8c7e018213026d35dc9bab480a21eb876838396556
+TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
`SOVERSION` guard check should be added if auto-update is enabled, although it seems that `harfbuzz` always uses `libharfbuzz.so.0`...

Reply:
harfbuzz did not break any previous API/ABI. For example, see Alpine Linux updates https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests?scope=all&state=merged&search=harfbuzz or any release notes of harfbuzz between v7 and v8.

Reply:
But no one can guarantee that they won't make changes in the future. I still think `SOVERSION` check is still necessary if auto-update is enabled.